recipes/boost/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "1.60.0" %}
+{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  {{ filename }}
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
+  md5: 65a840e1a0b13a558ff19eeb2c4f0cbe
+
+build:
+  features:
+    - vc9               # [win and py27]
+    - vc10              # [win and py34]
+    - vc14              # [win and py35]
+
+requirements:
+  build:
+    - gcc               # [linux]
+    - python
+    - icu               # [unix]
```

Comment:
I don't mind pinning, I would just like to have completely clear guidelines on when and why to do it.  I have seen things fail because versions were not pinned, and I've seen things fail because things were pinned.  Neither approach is completely right, but people should be making explicit, educated choices on these things.